src/model/Layer.ts

```diff
@@ -25,6 +25,11 @@ export interface DefaultLayerPropertyConfig {
   visible?: boolean;
 }
 
+export interface DownloadConfig {
+  downloadUrl: string;
+  formatName: string;
```

Comment:
```suggestion
  formatName?: string;
```

At least according to the JSON schema specified in SHOGun.

Reply:
good catch! fixed.